ParseAligns/abyss-fixmate.cc

```diff
@@ -488,7 +495,7 @@ int main(int argc, char* const* argv)
 
 	if (alignments.size() == sum) {
 		cerr << PROGRAM ": error: All reads are mateless. This "
-			"can happen when first and second read IDs do not match."
+			"can happen when first and second read IDs are not in same order."
```

Comment:
Order does not matter. Please revert this change.

Reply:
Done in 4348e7a3cc99e791370555ffff629395d313e258